src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -492,7 +477,7 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
                 {
                     auto scale_op_mem = instance.fused_memory(memory_offset);
                     dnnl::memory::desc desc = onednn::layout_to_memory_desc(scale_op_mem->get_layout(), dnnl::memory::format_tag::a, true);
-                    args.insert({DNNL_ARG_ATTR_OUTPUT_SCALES, scale_op_mem->get_onednn_memory(desc)});
+                    args.insert({DNNL_ARG_ATTR_SCALES | DNNL_ARG_DST, scale_op_mem->get_onednn_memory(desc)});
```

Comment:
is it used? I think it is not supposed to be called at this moment, right? If so, could you add exception here so that it is not called silently?

Reply:
Use dst scale in 3.0